internal/service/server/service.go

```diff
@@ -0,0 +1,22 @@
+package service
+
+import (
+	"log"
+
+	config "github.com/mrkovshik/yametrics/internal/config/server"
+	"github.com/mrkovshik/yametrics/internal/storage/server"
+)
+
+type Server struct {
+	Storage storage.IServerStorage
+	Logger  *log.Logger
```

Comment:
NICE TO HAVE: usually, `logger` is the last parameter.

Reply:
updated